src/script/cpp_api/s_base.h

```diff
@@ -158,6 +160,9 @@ class ScriptApiBase : protected LuaHelper {
 	void setGuiEngine(GUIEngine* guiengine) { m_guiengine = guiengine; }
 #endif
 
+	EmergeThread* getEmergeThread() { return m_emerge; }
+	void setEmergeThread(EmergeThread *emerge) { m_emerge = emerge; }
```

Comment:
This appears to be out of place here. Why not `s_mapgen.h`?

Reply:
It's indeed weird but that's how it works for other pointers too. :shrug: 
I'm not planning to rework this.